doc/source/whatsnew/v0.24.0.txt

```diff
@@ -297,7 +298,7 @@ MultiIndex
 I/O
 ^^^
 
--
+- :func:`read_html()` no longer ignores all-whitespace ``<tr>`` within ``<thead>`` when considering the ``skiprows`` and ``header`` arguments. Previously, users had to decrease their ``header`` and ``skiprows`` values on such tables to work around the issue. (:issue:`21641`)
```

Comment:
this is all ok, but would a subsection with a mini-example be instructive to the user about the revised functinaility? not saying 100% need to, but if a simple enough example (or even an image) would be helpful here

Reply:
I'm going to display my ignorance of the whatsnew format: ... where would this go?

Reply:
oh, you make a new subsection in the top somewhere, e.g. model after this

``Series and Index Data-Dtype Incompatibilities`` 

just add a new one below and you can make an extended example